### PR TITLE
docs(blog): add deployment action section to v1.30 blog

### PR DESCRIPTION
**Pull request by huanjani**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/blogs/release-v130.en.md [hunk 1]
```diff
@@ -39,6 +39,45 @@ Copilot v1.30 brings big enhancements to help you develop more flexibly and effi
 ## Ctrl-C
 
 ## Deployment actions
+Maybe you want to run a database migration before a workload deployment, and the workload's health check depends on the
+update. Or maybe you'd like your pipeline to execute some end-to-end or integration tests after a workload deployment. These actions are
+now possible with [Copilot pipelines](../docs/concepts/pipelines.en.md)!
```
Comment (Lou1415926) [inline site/content/blogs/release-v130.en.md:44]: This is just my personal thought maybe it's bad 🤣. I feel like "copilot" is implied by that this is a copilot blog post
```suggestion
now possible with [pipelines](../docs/concepts/pipelines.en.md)!  
```

File: site/content/blogs/release-v130.en.md [hunk 2]
```diff
@@ -39,6 +39,45 @@ Copilot v1.30 brings big enhancements to help you develop more flexibly and effi
 ## Ctrl-C
 
 ## Deployment actions
+Maybe you want to run a database migration before a workload deployment, and the workload's health check depends on the
+update. Or maybe you'd like your pipeline to execute some end-to-end or integration tests after a workload deployment. These actions are
+now possible with [Copilot pipelines](../docs/concepts/pipelines.en.md)!  
+
+While Copilot has supported 'test commands' for some time now, pre- and post-deployment actions extend pipeline functionality
```
Comment (Lou1415926) [inline site/content/blogs/release-v130.en.md]: nit!
```suggestion
While Copilot has supported 'test_commands' for some time now, pre- and post-deployment actions extend pipeline functionality 
```
```suggestion
While Copilot has supported ['test_commands'](https://aws.github.io/copilot-cli/docs/manifest/pipeline/#stages-test-cmds) for some time now, pre- and post-deployment actions extend pipeline functionality 
```

File: site/content/blogs/release-v130.en.md [hunk 3]
```diff
@@ -39,6 +39,45 @@ Copilot v1.30 brings big enhancements to help you develop more flexibly and effi
...
+
+While Copilot has supported 'test commands' for some time now, pre- and post-deployment actions extend pipeline functionality 
+and flexibility. For `test_commands`, Copilot inlines a buildspec with your command strings into the pipeline
+Cloudformation template; for `pre_deployments` and `post_deployments`, Copilot reads the [buildspec(s)](https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html)
+in your local workspace.  
+
+You control all the configuration for these actions right in your [pipeline manifest](../docs/manifest/pipeline.en.md). You may have multiple pre-deployment actions and multiple 
+post-deployment actions; for each one, you must provide the path to the [buildspec](https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html),
+relative to your project root, in the `[pre_/post_]deployments.buildspec` field. Copilot will generate a CodeBuild project for your action, deployed
+in the same region as the pipeline and app. Use Copilot commands within your buildspec to access the VPC of the environment
```
Comment (Lou1415926) [inline site/content/blogs/release-v130.en.md]:
```suggestion
in the same region as the pipeline and app. Use Copilot commands - such as `copilot svc exec` or `copilot task run` within your buildspec to access the VPC of the environment
```

File: site/content/blogs/release-v130.en.md [hunk 4]
```diff
@@ -39,6 +39,45 @@ Copilot v1.30 brings big enhancements to help you develop more flexibly and effi
...
+being deployed or deployed to; you may use the `$COPILOT_APPLICATION_NAME` and `$COPILOT_ENVIRONMENT_NAME` Copilot environment variables
+to reuse your buildspec file for multiple environments.
+
+You can even specify the deployment order of actions within your pre-deployment and
```
Comment (Lou1415926) [inline site/content/blogs/release-v130.en.md]:
```suggestion
You can even specify the order of actions within your pre-deployment and
```

File: site/content/blogs/release-v130.en.md [hunk 5]
```diff
@@ -39,6 +39,45 @@ Copilot v1.30 brings big enhancements to help you develop more flexibly and effi
...
+post-deployment groups by using the `depends_on` field; by default, the actions will deploy in parallel.
```
Comment (Lou1415926) [inline site/content/blogs/release-v130.en.md]: Not sure about "deploy", maybe "take place", "execute" or "run" 🤔 
```suggestion
post-deployment groups by using the `depends_on` field; by default, the actions will take place in parallel. 
```

**Review by Lou1415926 — APPROVED** [review]

File: site/content/blogs/release-v130.en.md (hunk 1 above)
Comment (huanjani) [reply site/content/blogs/release-v130.en.md:44]: I added that bc before it was possible with other pipelines but not copilot pipelines... 🤷🏼

**Review by dannyrandall — APPROVED** [review]

**Pull request merged**